### DISCUSSION TITLE
Please help with SQL query for provider reference JSON output.

### DISCUSSION THREAD

**Discussion by winnkathy-hub** [CMSgov/price-transparency-guide#747] [Q&A]
I'm trying to update my SQL query to follow the new 1.6.0 schema for provider reference.  I'm having issues understanding how to do this.  If anyone could help me with this, I would greatly appreciate it.

I'm attaching text document that shows the 4 fields in the SQL table and the 1.6.0 schema for provider reference.
[help with SQL.txt](https://github.com/CMSgov/price-transparency-guide/files/15358714/help.with.SQL.txt)
[help with SQL.txt](https://github.com/CMSgov/price-transparency-guide/files/15358717/help.with.SQL.txt)

**Comment by shaselton-usds** [comment]
@winnkathy-hub I'm not quite understanding this - my apologies. What is your SQL query actually trying to do? Your table looks pretty straightforward.
